owrx/source/hpsdr.py

```diff
@@ -51,11 +64,26 @@ def getName(self):
     def getInputs(self) -> List[Input]:
         return super().getInputs() + [
             RemoteInput(), 
-            NumberInput("rf_gain", "LNA Gain", "LNA gain between 0 (-12dB) and 60 (48dB)", validator=RangeValidator(0, 60)),
-            ]
+            NumberInput(
+                "rf_gain", 
+                "LNA Gain", 
+                "LNA gain between 0 (-12dB) and 60 (48dB) (default 20)", 
```

Comment:
Future consideration: I'm pretty sure users will appreciate if they don't have the math to do in their heads. Ideally, the input would accept the gain value directly and subtract -12 internally.